app/setup_handlers.go

```diff
@@ -38,6 +38,11 @@ func SetupHandlers(app *SifchainApp) {
 		// This is to make sure auth module migrates after staking
 		return app.mm.RunMigrations(ctx, app.configurator, newVM)
 	})
+	app.UpgradeKeeper.SetUpgradeHandler("0.10.0-rc.4", func(ctx sdk.Context, plan types.Plan, vm m.VersionMap) (m.VersionMap, error) {
```

Comment:
this might skip some of the migratios because we are not explicitly settng the version to 1 .
Is that the plan for this handler ?

Reply:
Well haven't those migrations already been run with rc.3? Setting to 1 is only for the first time a the new migration system is used right?